packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -68,6 +68,37 @@ void main() {
       expect(message, contains('Actual: ?:<exactly one widget with text "foo" (ignoring offstage widgets): Text("foo")>\n'));
       expect(message, contains('Which: means one was found but none were expected\n'));
     });
+
+    testWidgets('pumping', (WidgetTester tester) async {
+      await tester.pumpWidget(new Text('foo'));
+      int count;
+
+      AnimationController test = new AnimationController(duration: const Duration(milliseconds: 5100));
```

Comment:
To avoid having to make a call on whether if you pump and the time exactly lands on the duration the controller should finish right there and then or the controller should require one more pump to finish. (Basically, to avoid taking a position on whether the duration is > or >=, an exclusive bound or an inclusive bound.)